apps/rehabstod/src/store/sickLeaveApi.ts

```diff
@@ -109,6 +110,14 @@ const sickLeaveApi = api.injectEndpoints({
         }
       },
     }),
+    getRekoStatusForPatient: builder.query<RekoStatus, { patientId: string; endDate: string; startDate: string }>({
+      keepUnusedDataFor: 0,
```

Comment:
Perhaps we could use the cache instead? We could add a tag with specific identifier that gets invalidated with setRekoStatus.

setRekoStatus:
```typescript
invalidatesTags: (result, error, {patientId}) => [{type: 'RekoStatus', id: patientId}],
```

getRekoStatusForPatient:
```typescript
providesTags: (result, error, {patientId}) => [{ type: 'RekoStatus', id: patientId }],
```

Reply:
I remember Matias mentioned at a meeting that we shouldn't use the cache at all for the patientview.

Reply:
Alright, let's keep the `keepUnusedDataFor`. I think we still need to invalidate `getRekoStatusForPatient` when `setRekoStatus` gets called.

Reply:
This gives a ts error

Reply:
Probably missing 'RekoStatus' in the list of tags in `api.ts` ?